boe_min.sql

```diff
@@ -6,81 +6,80 @@
 ;
 
 CREATE TABLE IF NOT EXISTS minutes(
```

Comment:
Would still be helpful to explain each entity. For example, each row in the minutes table represents one meeting of the BoE.